Dockerfile

```diff
@@ -53,4 +79,4 @@ ENV TF_VAR_ansible_vars=/workspace/ansible-vars.yaml
 ENV ANSIBLE_CONFIG=/viya4-iac-k8s/ansible.cfg
 
 VOLUME ["/workspace"]
-ENTRYPOINT ["/viya4-iac-k8s/docker-entrypoint.sh"]
+ENTRYPOINT ["/viya4-iac-k8s/docker-entrypoint.sh"]
```

Comment:
File needs a correct line ending. This can be done in VS Code automatically. This usually happens when switching between windows/linux or your editor, i.e. nano or something strips the last line without adding the correct file ending.